spec/On_tactical_air_risk_mitigation.md

```diff
@@ -0,0 +1,164 @@
+# On tactical air risk mitigation
+
+Author(s): Kai Lothar John <https://github.com/jokalode>
+
+© 2023 by Kai Lothar John. ALL RIGHTS RESERVED.
```

Comment:
Hello @jokalode , Thank you for this and all the effort. As we had mentioned per the Dronecode policy we cannot accept copyrighted material, the best way to do this is to publish this article on your website and we can link it here.

Reply:
Fair point. I forgot about the copyright notice.

I will prepare a draft for the specification re. traffic information service free of copyright issues.